utils/airtable.test.jsx

```diff
@@ -0,0 +1,38 @@
+import '@testing-library/jest-dom';
+import findMatchingRecord from './findMatchingRecord';
+
+const fakeAirtableData = [
+  {
+    _rawJson: {
+      fields: {
+        'Component Name': 'Disclosure (Show/Hide)',
+        Slug: 'disclosure',
+        'Keyboard Interactions': 'a different set of bullet points',
+        'Usage Guidelines': 'some more different paragraphs',
+      },
+    },
+  },
+];
+
+describe('findMatchingRecord', () => {
+  it('returns all of the data associated with a particular component', () => {
+    const mockComponentName = 'disclosure';
+    const matchingRecord = findMatchingRecord(
+      fakeAirtableData,
+      mockComponentName,
+    );
+    // eslint-disable-next-line no-underscore-dangle
+    const expectedResult = fakeAirtableData[0]._rawJson.fields;
+    expect(matchingRecord).toStrictEqual(expectedResult);
+  });
+});
+describe('an unsuccessful API call', () => {
+  it('returns null when there is no match', async () => {
+    const mockComponentName = 'accordion';
+    const matchingRecord = findMatchingRecord(
+      fakeAirtableData,
+      mockComponentName,
+    );
+    expect(matchingRecord).toBeNull();
+  });
+});
```

Comment:
Normally I would put this `it` statement in the first `describe` block, since it is also testing the same function. For organization, it's often handy to have multiple tests grouped together like that.

```js
describe('someFunction', () => {
  it('handles the happy path', () => {});

  it('handles the sad path'. () => {});
});
```

That said, these are great tests!

Reply:
Easy fix! 
Does this look better? https://github.com/sparkbox/accessible-components/pull/34/commits/2c75dcfda2111c6bb615701ac4084aaff72919e8